cmd/client-fs.go

```diff
@@ -352,6 +351,19 @@ func (f *fsClient) Put(reader io.Reader, size int64, contentType string, progres
 	return totalWritten, nil
 }
 
+// Put - create a new file with metadata.
+func (f *fsClient) PutWithMetadata(reader io.Reader, size int64, metadata map[string][]string, progress io.Reader) (int64, *probe.Error) {
+	return f.put(reader, size, metadata, progress)
+}
+
+// Put - create a new file.
+func (f *fsClient) Put(reader io.Reader, size int64, contentType string, progress io.Reader) (int64, *probe.Error) {
+	metadata := map[string][]string{
+		"Content-Type": {contentType},
+	}
+	return f.put(reader, size, metadata, progress)
```

Comment:
We are not doing anything with metadata in FS, I think it is better to avoid passing metadata here, just for better code clarity.